content/raw/wallet/add_account.md

```diff
@@ -36,6 +36,15 @@ Users often want to add an account to their wallet and this is a very common use
 
 ### Generate an account
```

Comment:
Note: @carlfairclough is still working on these designs, when these designs are ready this spec should be reviewed and updated